### PR TITLE
Updated Travis CI config

### PR DESCRIPTION
**Pull request by cheba**

### CODE REVIEW

**Comment by packetmonkey** [comment]
I noticed that OPTS was mistyped ORTS in this travis config, can you fix that?

It also looks like 2.0.0p0 is broken, does anyone want to spend any time fixing it or should we just list 2.0.0p0 as unsupported? @jessedoyle you have  some recent commits, thoughts?

**Comment by cheba** [comment]
@packetmonkey Updated.

**Comment by jessedoyle** [comment]
@packetmonkey: I took a look at the `2.0.0p0` failures and they all seem related to Mocha's teardown mechanism.

I was able to find [this bug report](https://bugs.ruby-lang.org/issues/7843) for `2.0.0p0` that is likely the cause of the failures. As soon as I changed my Ruby version to `>= 2.0.0p195` (the bugfix was included in this version), the issues with Mocha seem to go away.

My suggestion is to drop support for `2.0.0p0` and test against `2.0.0p195` instead.

Alternatively, we could rewrite the failing tests to not use Mocha, but I'm fairly unfamiliar with `prawn-table`'s test suite.

**Comment by cheba** [comment]
@jessedoyle Dropping mocha isn't an option?

This gem is under Prawn org and is meant to work with the Prawn gem. Consequently it means it has to support all the same Ruby versions the Prawn gem does. I would consider changing testing strategy especially since the failure doesn't seem to come from the gem lib code.

Granted, that involves much more work than simply dropping 2.0.0-p0 support but isn't consistency worth it?

**Comment by packetmonkey** [comment]
@cheba I think the problem is now how much effort is it worth supporting an edge case on a library that isn't very well supported to begin with, given how this library is officially an ' experimental extraction'.

Which isn't to say I'll not merge a pr to fix it, but I would more easily mark 2.0.0-p0 as allowed failure in travis to get the CI green so the rest of the development can be done with what confidence the test suite can provide.

Basically I probably won't find the time to do that work, so if one of you are willing then great! Otherwise tweaking travis to be green by ignoring this one version is better than doing nothing I think.

Thoughts?

**Comment by jessedoyle** [comment]
> Granted, that involves much more work than simply dropping 2.0.0-p0 support but isn't consistency worth it?

We can definitely drop Mocha as a dependency, but that's a huge undertaking. If I remove Mocha from the test suite, 154 specs fail - which is more than half of the entire test suite. Rewriting those tests will require a significant investment of time from someone familiar with the codebase.

We could rewrite the 4 failing tests on `2.0.0p0` to not use Mocha, this will likely solve our issue on various MRI versions.

I'm not sure what's causing the JRuby `1.7.22` failures, but they also seem Mocha-related so there may be a bigger issue.

I would be able to rewrite the 4 failing tests on `2.0.0p0` to stop using Mocha. @packetmonkey Do you think that is worthwhile?

**Comment by packetmonkey** [comment]
Fixing the 4 failing tests for 2.0.0p0 would probably be the smallest change while preserving official 2.0.0p0 support, so if you are up for it I say give it a shot!

**Comment by packetmonkey** [comment]
Can we rebase this on master so we can avoid a merge conflict when we go to update those 4 tests?

**Comment by dhurba87** [comment]
@pointlessone, an image is not working for version 2.3.3 and 2.3.1. My old ruby version was 2.3.0 and it was working well but when I upgrade ruby version to 2.3.3 it stopped displaying images.

**Comment by pointlessone** [comment]
@dhurba87 Could you please open a new issue with a detailed description and reproduction steps?
